### ISSUE TITLE
HTTPRoute in Viz dashboard

### ISSUE THREAD

**Issue by Sierra1011** [linkerd/linkerd2#11088]
### What problem are you trying to solve?

The indication given by the documentation is that `TrafficSplit`s are dead, long live the `HTTPRoute`. Which is great, happy to fall in line. However there's no easy visibility of the `HTTPRoute` config from the LinkerD Viz dashboard, like there currently is for `TrafficSplit`s.

### How should the problem be solved?

It'd be great if this could be added alongside the existing `TrafficSplit` option!

### Any alternatives you've considered?

I'm not entirely sure how else to visualise the split of traffic across services. Maybe something in the TAP page would go towards it.

### How would users interact with this feature?

Just clicking on the button under Configuration and viewing the parent `HTTPRoute` object, with the backend services and the relative weightings. 
You could also do a cute line graph at the top, maybe as a stacked area chart with the available history scrolling across it. That's hugely out of scope, I know, and I'd very happily settle for a single entry that says the parent and dependent services with the weights!

### Would you like to work on this feature?

maybe

**Comment by Sierra1011** [comment]
There is still a need for this feature, and I unfortunately haven't been able to prioritise it.
TL;DR: Bump

**Comment by kflynn** [comment]
There's no timeline for removing support for TrafficSplits, so it's probably a bit premature to call them dead (they're just _pining_?) *but* that said, yes, we are quite interested in support for HTTPRoutes in Linkerd Viz. 🙂
